README.md

```diff
@@ -24,3 +24,22 @@ Import into HTML:
 Import into Dart:
 
     import 'package:polymer_elements/iron_input.dart';
+
+### platinum-elements
+
+For `platinum-sw-register` to work properly the following requirements should be met:
+
+ 1. you have to provide an `ws-import.js`
+   script in the root of your application containing only the following line:
+
+
+    importScripts('packages/polymer_elements/src/platinum-sw/service-worker.js');
```

Comment:
If you are talking about `ws-import.js` this is something that the user should provide by himself in the root folder of the app and cannot be distributore with the library. See https://github.com/PolymerElements/platinum-sw/blob/master/README.md#top-level-sw-importjs for details